scripts/obmd.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=OBM daemon for HIL
+After=network.target
+After=postgresql.service
+
+[Service]
+Type=simple
+User=hil
```

Comment:
It might be better to have obmd run as a separate user. This way hil and obmd can't see each-other's configs/databases, which would be better, since they shouldn't need to.